src/olympia/addons/models.py

```diff
@@ -360,25 +361,25 @@ class Addon(OnChangeMixin, ModelBase):
                   'A value of 0 has no impact')
     requires_payment = models.BooleanField(default=False)
 
-    # The order of those managers is very important:
-    # The first one discovered, if it has "use_for_related_fields = True"
-    # (which it has if it's inheriting `ManagerBase`), will
-    # be used for relations like `version.addon`. We thus want one that is NOT
-    # filtered in any case, we don't want a 500 if the addon is not found
-    # (because it has the status amo.STATUS_DELETED for example).
-    # The CLASS of the first one discovered will also be used for "many to many
-    # relations" like `collection.addons`. In that case, we do want the
-    # filtered version by default, to make sure we're not displaying stuff by
-    # mistake. You thus want the CLASS of the first one to be filtered by
-    # default.
-    # We don't control the instantiation, but AddonManager sets include_deleted
-    # to False by default, so filtering is enabled by default. This is also why
-    # it's not repeated for 'objects' below.
     unfiltered = AddonManager(include_deleted=True)
     objects = AddonManager()
 
     class Meta:
         db_table = 'addons'
+        # This is very important:
+        # The default base manager will be used for relations like
+        # `version.addon`. We thus want one that is NOT filtered in any case,
+        # we don't want a 500 if the addon is not found (because it has the
+        # status amo.STATUS_DELETED for example).
+        # The CLASS of the first one discovered will also be used for "many to
+        # many relations" like `collection.addons`. In that case, we do want
+        # the filtered version by default, to make sure we're not displaying
+        # stuff by mistake. You thus want the CLASS of the first one to be
+        # filtered by default.
+        # We don't control the instantiation, but AddonManager sets
+        # include_deleted to False by default, so filtering is enabled by
+        # default. This is also why it's not repeated for 'objects' below.
```

Comment:
This needs rewording now - there is only one manager below.  `objects` is only above.

Reply:
Indeed, I updated it in 92abe1e2ba, does that work better?